src/components/Team/index.js

```diff
@@ -29,6 +30,20 @@ const TEAM_HEADLINERS = [
 ];
 
```

Comment:
You can simplify the codebase by defining the `gradient` outside:
```
const gradient = 'linear-gradient(0deg, #090909 0%, rgba(9, 9, 9, 0) 100%)';
```
and then using it in `style` function as in suggestion below